tests/test_init.py

```diff
@@ -0,0 +1,18 @@
+import inspect
+
+import globus_automate_client
+
+
+def test_all_exports():
```

Comment:
This test seems like a tautology? In which cases could something appear in `__all__` yet not appear under the import or vice versa?

Reply:
`__all__` isn't checked by Python unless you explicitly use `from blah import *`. Therefore this test ensures that future development both _imports_ what's required by `__all__` and _exports_ what was almost certainly imported for the purpose. For example:


```python
from submodule1 import ExistingThing
from submodule2 import KurtImportedMe

__all__ = (
    "ExistingThing",
    "KurtImportedMeButCannotCopyAndPasteCorrectly",
)
```

You can import that file using `import blah` without any issue, but you'll get an unexpected `AttributeError` if you use `from blah import *`.

Reply:
I'm going to mark this as resolved but please reopen as needed!